src/App/App.js

```diff
@@ -1,81 +1,47 @@
 import './App.css';
 import React, { Component } from 'react';
-import Header from '../Header/Header';
 import CardContainer from '../CardContainer/CardContainer';
 import MovieDisplay from '../MovieDisplay/MovieDisplay';
 import ErrorDisplay from '../ErrorDisplay/ErrorDisplay';
-import { fetchData } from '../util.js'
-
+import { fetchData } from '../util.js';
+import { Route, Switch } from 'react-router-dom';
 
 class App extends Component {
  constructor(){
    super();
    this.state = {
      movies: [],
-     currentSelectionId: null,
-     error: null,
-     selectedMovie: null
+     error: null
    }
  }
 
  componentDidMount = () => {
   fetchData('movies').then(data => {
     this.setState({movies: [...this.state.movies, ...data.movies]})
   }).catch((data) => this.setState({error: data.message}))
-  // The above .catch is setting our state, however the react dev tools simply shows it as an empty object. We should revisit this after we make our error display component. 
```

Comment:
It is amazing how much less code is needed now that we've refactored for Router!